project/infra/code-index/src/cli.py

```
@@ -0,0 +1,260 @@
+console = Console()
+
+
+@app.command()
+def index(
+    source_path: Path = typer.Argument(
+        ...,
+        help="Path to source code directory",
+        exists=True,
+        file_okay=False,
+        dir_okay=True,
+    ),
```

Ruff (0.15.2)

[warning] Do not perform function call `typer.Argument` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)